docs/securing_client_connection.rst

```diff
@@ -188,6 +190,8 @@ follows:
     from hazelcast.config import SSLProtocol
 
     client = hazelcast.HazelcastClient(
+        # You can also set this to "TLSv1_3"
```

Comment:
nit: this fact was already mentioned above, so this comment could be removed for the sake of brevity